src/Hazelcast.Net/Clustering/Heartbeat.cs

```diff
@@ -122,7 +112,7 @@ private async Task RunAsync(CancellationToken cancellationToken)
         {
             _logger.LogDebug("Run heartbeat");
 
-            var now = DateTime.UtcNow;
+            var now = DateTime.Now; // now, or utcNow, but *must* be same as what is used in socket connection base!
```

Comment:
Can we refactor out the `DataTime.UtcNow` usage altogether?
(Maybe just the static EPOC date)

Reply:
What about just using the internal Clock class to make them same everywhere?

Reply:
looking into it